examples/hello-compute/main.rs

```diff
@@ -12,8 +12,10 @@ async fn run() {
             .collect()
     };
 
-    // To see the output, run `RUST_LOG=info cargo run --example hello-compute`.
-    log::info!("Times: {:?}", execute_gpu(numbers).await);
+    let times = execute_gpu(numbers).await;
+    println!("Times: {:?}", times);
```

Comment:
IMO it's slightly distracting to special-case wasm with `cfg` here  – I'd prefer to force the log level in `main` or pass a logging function to `run` if we want to ensure it's displayed by default

Reply:
ok, sure. I just got confused by the lack of output when running it